fiftyone/utils/depth_anything.py

```
@@ -0,0 +1,316 @@
+                :class:`PIL.Image.Image`, torch Tensor, or numpy array
+
+        Returns:
+            a tuple of ``(img_array, original_size)``
+        """
+        from PIL import Image as PILImage
+
+        if isinstance(img, str):
+            import os
+
+            if not os.path.isfile(img):
+                raise ValueError("Image file not found: %s" % img)
```

Ruff (0.14.10)

Use format specifiers instead of percent format

Replace with format specifiers

(UP031)